src/modules/users/users.service.ts

```diff
@@ -0,0 +1,91 @@
+import {
+  Injectable,
+  ConflictException,
+  NotFoundException,
+} from '@nestjs/common';
+import { CreateUserDto, UpdateUserDto } from './dto/user.dto';
+import { DatabaseService } from '../database/database.service';
+
+@Injectable()
+export class UsersService {
+  constructor(private database: DatabaseService) {}
+  async create(createUserDto: CreateUserDto) {
+    try {
+      const newUser = await this.database.createUser(createUserDto);
```

Comment:
this, as mentioned before, should be used like "this.databaseService.user.create({data: createUserDto})" and not calling the method that will be removed after this review